htdocs/stripe/admin/stripe.php

```diff
@@ -272,16 +273,36 @@
 	print '</td></tr>';
 }
 
-// Minimal amount for force 3Dsecure if it's optionnal
+// Activate Payment Request API
+if ($conf->global->MAIN_FEATURES_LEVEL >= 2)	// TODO Not used by current code
+{
+  print '<tr class="oddeven"><td>';
+	print $langs->trans("STRIPE_PAYMENT_REQUEST_API").'</td><td>';
+  if ($conf->use_javascript_ajax) {
+    print ajax_constantonoff('STRIPE_PAYMENT_REQUEST_API');
+  } else {
+    $arrval = array('0' => $langs->trans("No"), '1' => $langs->trans("Yes"));
+    print $form->selectarray("STRIPE_PAYMENT_REQUEST_API", $arrval, $conf->global->STRIPE_PAYMENT_REQUEST_API);
+  }
+	print '</td></tr>';
+}
+
+// Activate SEPA DIRECT_DEBIT
 if ($conf->global->MAIN_FEATURES_LEVEL >= 2)	// TODO Not used by current code
 {
 	print '<tr class="oddeven"><td>';
-	print $langs->trans("STRIPE_MINIMAL_3DSECURE").'</td><td>';
-	print '<input class="flat" name="STRIPE_MINIMAL_3DSECURE" size="3" value="' .$conf->global->STRIPE_MINIMAL_3DSECURE . '">'.$langs->getCurrencySymbol($conf->currency).'</td></tr>';
+	print $langs->trans("STRIPE_SEPA_DIRECT_DEBIT").'</td><td>';
+  if ($conf->use_javascript_ajax) {
+    print ajax_constantonoff('STRIPE_SEPA_DIRECT_DEBIT');
+  } else {
+    $arrval = array('0' => $langs->trans("No"), '1' => $langs->trans("Yes"));
+    print $form->selectarray("STRIPE_SEPA_DIRECT_DEBIT", $arrval, $conf->global->STRIPE_SEPA_DIRECT_DEBIT);
+  }
+	print '</td></tr>';
 }
 
 // Warehouse for automatic decrement
-if ($conf->global->MAIN_FEATURES_LEVEL >= 2)	// What is this for ?
+if ($conf->global->MAIN_FEATURES_LEVEL >= 2)	// warehouse to reduce stock for online payment
```

Comment:
A stock is decrease when a shipment is done, or when an invoice is validated (see setup of stock module). 
Never on a payment. so why this ?

Reply:
it was for choose warehouse on order's payment if this decrease option has been choosen but we can do it better. i have disabled / comment it.